ui/src/shared/components/cells/Cells.tsx

```diff
@@ -119,5 +142,13 @@ class Cells extends Component<Props & WithRouterProps> {
     }
   }
 }
-
-export default withRouter(Cells)
+const mstp = (state: AppState): StateProps => {
+  return {state}
```

Comment:
Why are we returning the entire state as state here? Which properties are we trying to access?

Reply:
Because we want the entire state? I dont think we can do anything but pass the entire state in

Reply:
we don't have to pass anything at all. It doesn't seem like we're actually using the redux state or actions to update any specific logic